next/modules/cards-and-rows/NoticeCard.tsx

```diff
@@ -26,12 +26,11 @@ const NoticeCard = ({ notice }: NoticeCardProps) => {
   return (
     <div className="group/showMore relative flex h-full w-full shrink-0 flex-col justify-between">
       <div className="flex h-full flex-col">
-        {/* TODO: Replace with MImage */}
-        <img
-          className="mb-4 h-48 w-full object-cover"
-          alt=""
-          src={image?.attributes?.url || Placeholder.src}
-        />
+        {image?.attributes && (
+          <div className="relative h-[163px] w-[271px] object-cover">
+            <MImage image={image?.attributes} fill />
+          </div>
+        )}
```

Comment:
This will not show Placeholder if no image is provided...

See also comments above.